app/components/TextInputWithDatePicker/TextInputWithDatePicker.jsx

```diff
@@ -141,26 +141,24 @@ export default class TextInputWithDatePicker extends React.Component {
     const {value} = event.target
     const newDate = new DateTime(event.target.value, format)
 
-    if (value.length > 0) {
```

Comment:
Make sure you have 'hide whitespace changes' checked, otherwise the diff looks more complicated than it is :slightly_smiling_face: